kafka-ui-react-app/src/components/common/NewTable/Table.tsx

```diff
@@ -198,97 +198,95 @@ const Table: React.FC<TableProps<any>> = ({
+            <React.Fragment key={row.id}>
+              <S.Row
+                expanded={row.getIsExpanded()}
+                onClick={handleRowClick(row)}
+                clickable={
+                  !enableRowSelection &&
+                  (row.getCanExpand() || onRowClick !== undefined)
+                }
+              >
                 {!!enableRowSelection && (
-                  <S.Th key={`${headerGroup.id}-select`}>
+                  <td key={`${row.id}-select`} style={{ width: '1px' }}>
                     {flexRender(
-                      SelectRowHeader,
-                      headerGroup.headers[0].getContext()
+                      SelectRowCell,
+                      row.getVisibleCells()[0].getContext()
                     )}
-                  </S.Th>
+                  </td>
                 )}
                 {table.getCanSomeRowsExpand() && (
-                  <S.Th expander key={`${headerGroup.id}-expander`} />
+                  <td key={`${row.id}-expander`} style={{ width: '1px' }}>
+                    {flexRender(
+                      ExpanderCell,
+                      row.getVisibleCells()[0].getContext()
+                    )}
+                  </td>
                 )}
-                {headerGroup.headers.map((header) => (
-                  <S.Th
-                    key={header.id}
-                    colSpan={header.colSpan}
-                    sortable={header.column.getCanSort()}
-                    sortOrder={header.column.getIsSorted()}
-                    onClick={header.column.getToggleSortingHandler()}
-                  >
-                    <div>
-                      {flexRender(
-                        header.column.columnDef.header,
-                        header.getContext()
-                      )}
-                    </div>
-                  </S.Th>
-                ))}
-              </tr>
-            ))}
-          </thead>
-          <tbody>
-            {table.getRowModel().rows.map((row) => (
-              <React.Fragment key={row.id}>
-                <S.Row
-                  expanded={row.getIsExpanded()}
-                  onClick={handleRowClick(row)}
-                  clickable={
-                    !enableRowSelection &&
-                    (row.getCanExpand() || onRowClick !== undefined)
-                  }
-                >
-                  {!!enableRowSelection && (
-                    <td key={`${row.id}-select`} style={{ width: '1px' }}>
-                      {flexRender(
-                        SelectRowCell,
-                        row.getVisibleCells()[0].getContext()
-                      )}
+                {row
+                  .getVisibleCells()
+                  .map(({ id, getContext, column: { columnDef } }) => (
+                    <td key={id} style={columnDef.meta}>
+                      {flexRender(columnDef.cell, getContext())}
                     </td>
-                  )}
```

Comment:
i think this change will effect all the tables that expand in our application, cause this `Table` component is used on every page and changing it will effect everywhere.

Reply:
@Mgrdich  Yeap, I deleted `TableWrapper` because it add `overflow-x: auto` property. It can create bug with dropdown for all table components.If dropdown size is larger than table size then scroll exists in `TableWrapper` See screens. I think the changes in git are because of the spaces. You can pull branch and compare with previous commit.

With `TableWrapper`
![Screenshot from 2022-12-04 08-12-05](https://user-images.githubusercontent.com/110557519/206890134-73b47a70-9099-4122-bc50-e3561def059b.png)
Without `TableWrapper`
![Screenshot from 2022-12-04 08-09-13](https://user-images.githubusercontent.com/110557519/206890271-aa1b963b-5f02-4edf-99e6-ae816b85a28b.png)

Reply:
I will check it out.